deployment/ansible/group_vars/all

```diff
@@ -28,3 +28,7 @@ postgis_version: "2.1"
 postgis_package_version: "2.1.*.pgdg14.04+1"
 
 nodejs_npm_version: 2.1.17
+
+sqs_queue: "raster-foundry"
```

Comment:
I personally try to keep things out of `group_vars/all` that don't apply to all environments. Mostly just to help reason about variable precedence across environments.

Can you please create a new file in `group_vars` named `packer` with these three variables? I can deal with making sure they wire up later.